examples/build.gradle

```diff
@@ -5,32 +5,31 @@ buildscript {
         google()
     }
     dependencies {
-        classpath("com.google.protobuf:protobuf-gradle-plugin:0.8.18")
+        classpath("com.google.protobuf:protobuf-gradle-plugin:${protobufGradlePluginVersion}")
     }
 }
 
 plugins {
     id "java"
-    id "com.google.protobuf" version "0.8.18"
+    id "java-library"
```

Comment:
I think java-library could be removed?

Reply:
does not work: I do not know why but we need bot `application` and` java-library` plugins.

Reply:
OK, we can see it later if it is relevant